process/block/cutoff/blockProcessingCutoffHandler.go

```diff
@@ -0,0 +1,130 @@
+package cutoff
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-go/common"
+	"github.com/multiversx/mx-chain-go/config"
+	logger "github.com/multiversx/mx-chain-logger-go"
+)
+
+var log = logger.GetOrCreate("process/block/cutoff")
+
+type blockProcessingCutoffHandler struct {
+	config config.BlockProcessingCutoffConfig
+}
+
+// NewBlockProcessingCutoffHandler will return a new instance of blockProcessingCutoffHandler
+func NewBlockProcessingCutoffHandler(cfg config.BlockProcessingCutoffConfig) (*blockProcessingCutoffHandler, error) {
+	err := checkConfig(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	return &blockProcessingCutoffHandler{
```

Comment:
we need a log.Warn on this constructor or on the factory stating the parameters at which the node will stop.

Reply:
added